rnd/autogpt_builder/src/components/ui/input.tsx

```diff
@@ -16,6 +25,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
           className,
         )}
         ref={ref}
+        defaultValue={value}
```

Comment:
We are not force-controlling the input, but if the value changes, we will update the ref.value

Reply:
This will allow to do semi-controlled input (e.g: using onBlur instead of onChange).